gulpfile.js

```diff
@@ -188,7 +188,7 @@ gulp.task('create-new-tag', (cb) => {
 
 gulp.task('build', ['static', 'test'])
 
-gulp.task('npm-publish', gulpShell.task('npm publish'));
+gulp.task('npm-publish', gulpShell.task('npm publish --access public'));
```

Comment:
You edited the wrong `gulpfile.js`! Should be the one in `app/templates/_gulpfile.js`